### PR TITLE
No csd linux by default

### PR DESCRIPTION
**Pull request by sbatten**

### CODE REVIEW

**Review by bpasero — APPROVED** [review]
@sbatten LGTM. But please open a test plan item and assign me for it 👍

File: src/vs/platform/windows/common/windows.ts [hunk 1]
```diff
@@ -265,6 +265,11 @@ export function getTitleBarStyle(configurationService: IConfigurationService, en
 		return 'native'; // not enabled when developing due to https://github.com/electron/electron/issues/3647
 	}
 
+	const accessibilityMode = configurationService.getValue('editor.accessibilitySupport');
```
Comment (bpasero) [inline src/vs/platform/windows/common/windows.ts]: @sbatten get this setting only when `!isMacintosh` ?

File: src/vs/platform/windows/common/windows.ts [hunk 2]
```diff
@@ -265,6 +265,11 @@ export function getTitleBarStyle(configurationService: IConfigurationService, en
...
+	if (!isMacintosh && accessibilityMode) {
```
Comment (bpasero) [inline src/vs/platform/windows/common/windows.ts]: @sbatten the value of `editor.accessibilitySupport` is a string, so how does your check here actually work?

File: src/vs/workbench/browser/parts/titlebar/menubarControl.ts [hunk 3]
```diff
@@ -203,39 +199,41 @@ export class MenubarControl extends Disposable {
...
-					run: () => {
-						window.open('https://go.microsoft.com/fwlink/?linkid=2038566');
-					}
-				},
-				{
-					label: nls.localize('neverShowAgain', "Don't Show Again"),
-					run: () => {
-						this.storageService.store('menubar/electronFixRecommended', true, StorageScope.GLOBAL);
-					}
+		const message = nls.localize('menubar.linuxTitlebarRevertNotification', "We have updated the default title bar on Linux to use the native setting. If you prefer, you can go back to the custom setting ");
```
Comment (bpasero) [inline src/vs/workbench/browser/parts/titlebar/menubarControl.ts]: @sbatten  why is there a trailing whitespace at the end of the sentence (`you can go back to the custom setting `)?

**Review by bpasero — CHANGES_REQUESTED** [review]
@sbatten feedback provided

@kieferrm maybe on the wording of the notification, which currently is: "We have updated the default title bar on Linux to use the native setting. If you prefer, you can go back to the custom setting".

**Review by bpasero — APPROVED** [review]
@sbatten looks like you changed to show separate notifications now. LGTM with some polish:
* "Go to Setting" => "Open Settings"
* it looks like you always store the "Do not show again" state after showing the notification, shouldn't that be a decision that is made only by the user when clicking "Don't Show Again"?

**Comment by bpasero** [comment]
@sbatten also, I would inline the "More Info" to be a link within the notification and not have it as extra button to reduce clutter. You can use markdown style styntax for that.

**Comment by sbatten** [comment]
@bpasero, updated to use inline link as suggested and switched back to Open Settings for consistency. I kept the behavior to not keep  showing this message even without the user's input so as to not but the user. The notifications are not critical unlike the previous notification to address an electron bug.

**Comment by bpasero** [comment]
@sbatten LGTM

**Pull request merged**